test/factories/event_log.rb

```diff
@@ -1,3 +1,6 @@
 FactoryBot.define do
-  factory :event_log
+  factory :event_log do
+    event_id { EventLog::NO_SUCH_ACCOUNT_LOGIN.id }
+    uid { create(:user).id }
```

Comment:
It'd be more accurate (as documentation) to use the user's `uid` instead of the integer `id` here

Reply:
Ah - good idea! 👍

Reply:
Addressed in https://github.com/alphagov/signon/pull/2331/commits/4405fad9436d5eeb62412e25197e027d1701f32a